scm-core/src/main/java/sonia/scm/web/UserAgent.java

```diff
@@ -56,10 +56,26 @@
    */
   private UserAgent(String name, boolean browser,
     Charset basicAuthenticationCharset)
+  {
+    this(name, browser, basicAuthenticationCharset, false);
+  }
+
+  /**
+   * Constructs a new user agent
+   *
+   *
+   * @param name
+   * @param browser
+   * @param basicAuthenticationCharset
+   */
+  private UserAgent(String name, boolean browser,
+                    Charset basicAuthenticationCharset,
+                    boolean scmClient)
```

Comment:
I don't think we should keep the old constructor. This does not make the handling of the booleans better, and this is private and therefore internal, only.
And by the way it's dead code, too.